src/tasks/netSdk/NetSdkRunTaskProvider.ts

```diff
@@ -55,4 +51,55 @@ export class NetSdkRunTaskProvider extends DockerTaskProvider {
 
         return Promise.resolve();
     }
+
+    public createNetSdkRunTask(options?: Omit<NetSdkRunTask, "type">): { task: Task, promise: Promise<number> } {
+        let task: Task;
+        const definition = {
+            ...options,
+            type: NetSdkRunTaskType,
+        };
+
+        const promise = new Promise<number>((resolve, reject) => {
+            task = new Task(
+                definition,
+                this.getProjectFolderFromTask(definition),
```

Comment:
Folder in relation to tasks is confusing, but it specifically refers to Workspace folders, which are the root working folder you have open in vscode. Technically you can have multiple workspaces open as separate roots, but TaskScope.Workspace indicates a task applies to the current active workspace.

```suggestion
                TaskScope.Workspace,
```